test/regression/cases/file_tree/experiment.yaml

```diff
@@ -28,3 +29,9 @@ target:
     DD_PROFILING_WAIT_PROFILE: true
 
     DD_INTERNAL_PROFILING_EXTRA_TAGS: experiment:file_tree
+
+    # System Probe
+    DD_SYSTEM_PROBE_INTERNAL_PROFILING_ENABLED: true
```

Comment:
Is this change expected?

Reply:
@scottopell added those to debug some weird values we were seeing in the SMP check. It turns out the problem was that system probe was segfaulting 😄 

We fix the segfault here https://github.com/DataDog/datadog-agent/pull/27195/commits/e29252c7a36c5c9feb6ce6725ca582b3c3ed0bb2